course4/homework1/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+project(homework1)
+include_directories(.)
```

Comment:
I use visual studio. 
project(homework1) creates a solution file for me to open up using vs.
The second line I agree I should delete.